trip_app/core/utils.py

```diff
@@ -19,18 +23,40 @@ def raise_for_status(resp):
         raise Exception(http_error_msg)
 
 
-def str_to_geopoint(data):
+def str_to_geopoint_or_validation_error(data):
```

Comment:
Yeah, you can specify the error case in the docstring (`:raises | Raises` depending on the docstring style). The short name makes more sense.

Reply:
I think that it is rather important information to include it in a method name, though it seems rather cumbersome it can make a life easier to the client code, and save time on debugging.
But perhaps it is just my passion to too long explicit names... I will change it.

Reply:
Thanks guys! This should be updated now.

Reply:
You can't place every meaning into the name, so it's better to stick with just the key meaning. Docstrings matter a lot though.